jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -202,10 +205,12 @@ YUI.add('entity-details', function() {
 
     render: function() {
       return (
-        <div className={this._generateClasses()}>
+        <div className={this._generateClasses()}
+          ref="content"
+          tabIndex="0">
```

Comment:
Does this mess with any of the accessibility work we've done for the breadcrumb?